docs/gen_images.py

```diff
@@ -107,7 +107,7 @@ def generate_hierarchy_image() -> None:
     lost,
     dict(
         styling.BLUE_ACTOR_FNCS,
-        **{"junction": lambda _: {"fill": diagram.RGB(220, 20, 60)}},  # type: ignore
+        **{"junction": lambda _, __: {"fill": diagram.RGB(220, 20, 60)}},  # type: ignore
```

Comment:
Unnecessary dict unpacking and type: ignore.

```suggestion
        junction=lambda _, __: {"fill": diagram.RGB(220, 20, 60)},
```

By the way, it seems pylint isn't configured properly to warn about too long lines.